### ISSUE TITLE
Crashing on CLLayer position contains NaN

### ISSUE THREAD

**Issue by nnhubbard** [nfarina/calloutview#105]
I have been getting some crash reports that show the following:

**Comment by nnhubbard** [comment]
I actually added the following check, but I still get the error:

```obj-c
// now set the *actual* anchor point for our layer so that our "popup" animation starts from this point.
    if (isnan(anchorX) || isnan(anchorY)) {
        return;
    }
    CGPoint anchorPoint = [layer convertPoint:CGPointMake(anchorX, anchorY) toLayer:self.layer];
```

**Comment by nfarina** [comment]
Hm that's weird - can you reproduce this issue in the sample project?

**Comment by nnhubbard** [comment]
No I can't. I just get crash logs through Crashlytics about it. I have been getting them since we launched our app in December.

**Comment by grusha** [comment]
Hi! I get these crashes as well. 
@nnhubbard Have you found what's causing it?

**Comment by nnhubbard** [comment]
@grusha No, have never figured it out. I still get the reports on Crashlytics but cannot reproduce.

**Comment by a8695363** [comment]
> @grusha No, have never figured it out. I still get the reports on Crashlytics but cannot reproduce.

HI,Have you found what's causing it?

**Comment by a8695363** [comment]
> @grusha No, have never figured it out. I still get the reports on Crashlytics but cannot reproduce.



> No I can't. I just get crash logs through Crashlytics about it. I have been getting them since we launched our app in December.



> @grusha No, have never figured it out. I still get the reports on Crashlytics but cannot reproduce.

@nnhubbard HI,Have you found what's causing it now?

**Comment by nnhubbard** [comment]
No, still never found the cause.
